Main.cpp

```diff
@@ -191,43 +239,63 @@ int main() {
       // canBus.printRxAll();
     }
 
-    chThdSleepMilliseconds(50);
+    chThdSleepMilliseconds(10000);
   }
 }
 
-// module utility functions
 // @brief Convert bytes in RX buffer from ADC reading to temperature
 // @return temp Temperature in degrees C, resolution of .25
+// TODO: Add in per-channel moving average
 uint8_t adc_to_temp(uint8_t* rxbuf) {
   static uint16_t raw{};
-  static int norm{};
+  static double voltage{};
```

Comment:
I meant to prevent re-allocation of these frequently used vars (repeatedly calling this function 28 times for each iteration of the calling thread's while loop) by making them static. Are you saying that this is optimized away?